types/k6/experimental/browser.d.ts

```diff
@@ -2454,6 +2501,29 @@ export interface Page {
      */
     mouse: Mouse;
 
+    /**
+     * Emitted when JavaScript within the page calls one of console API methods
+     * , e.g. `console.log` or `console.dir`. Also emitted if the page throws
+     * an error or a warning.
+     *
+     * The arguments passed into `console.log` are available on the
+     * {@link ConsoleMessage} event handler argument.
+     *
+     * **Usage**
+     *
+     * ```js
+     * page.on('console', async msg => {
+     *   const values = [];
+     *   for (const arg of msg.args())
+     *     values.push(await arg.jsonValue());
+     *   console.log(...values);
+     * });
+     * await page.evaluate(() => console.log('hello', 5, { foo: 'bar' }));
```

Comment:
These `async` and `await` keywords look incorrect for how it's used in k6 browser, no? What happens if we don't work with them?

E.g. `page.evaluate` doesn't return a `Promise`, and the `listener` returns a `void` not a `Promise<void>`.

Reply:
Right! I got inspiration from other sources and overlooked this example :)
Fixed in [c3dc7e9](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66549/commits/c3dc7e936276dd587026f5c90d9aba14b3f0d226).